robot/rospackages/src/mcu_control/scripts/NavigationNode.py

```diff
@@ -34,6 +38,10 @@ def subscriber_callback(message):
         # note that direction is based on compass directions where E is 90 and W is -90
         msg.x = Direction_adjust
         msg.y = Rov_to_des_distance
+        if(msg.y < 50):
+            rospy.loginfo('moving on to next coorinates')
```

Comment:
why 50?

Reply:
Oh yeah, I should have mentioned this somewhere else, but this was left there for easy debugging purposes. It switches to new coordinates when the distance is less than a certain number, but in real purposes I'm not sure what that number should be, if the distance is in meters then it should probably less than 0.5 or something (I'd have to ask Josh) but for testing purposes it would be a pain if I had it was that low.